core/storage/job/gae_models.py

```diff
@@ -94,17 +112,39 @@ def get_recent_jobs(cls, limit, recency_msec):
 
     @classmethod
     def get_all_unfinished_jobs(cls, limit):
+        """Gets limit number of unfinished jobs.
```

Comment:
`Gets at most ``limit`` unfinished jobs.`

(but put just a single backtick around limit, not two -- github markdown doesn't seem to let me do that).